Visual Studio Code (1.77.1, attached-container, desktop)
Jupyter Extension Version: 2023.3.1000892223.
Python Extension Version: 2023.6.0.
Workspace folder /home/cms-jovyan
info 04:46:03.786: ZMQ loaded via fallback mechanism.
User belongs to experiment group 'jupyterEnhancedDataViewer'
User belongs to experiment group 'jupyterTest'
info 04:46:04.089: LSP Notebooks experiment is enabled
info 04:46:04.561: Checking for server existence.
info 04:46:04.562: Connecting to server
info 04:46:04.563: Connecting remote server kernel http://127.0.0.1:8888/
Connecting to Jupyter server at http://127.0.0.1:8888/
info 04:46:04.563: Creating server with url : http://127.0.0.1:8888/
info 04:46:04.584: Server started.
info 04:46:04.584: Creating server with url : http://127.0.0.1:8888/
info 04:46:04.596: Checking for server existence.
info 04:46:04.597: Connecting to server
info 04:46:04.598: Connecting remote server kernel http://128.205.11.158:8882/
Connecting to Jupyter server at http://128.205.11.158:8882/
info 04:46:04.598: Creating server with url : http://128.205.11.158:8882/
info 04:46:04.600: Server started.
info 04:46:04.600: Creating server with url : http://128.205.11.158:8882/
info 04:46:05.017: Starting Jupyter Session startUsingPythonInterpreter, .jvsc74a57bd0916dbcbb3f70747c44a77c7bcd40155683ae19c65e1c03b4aa3499c5328201f1./bin/python3./bin/python3.-m#ipykernel_launcher (Python Path: , EnvType: Unknown, EnvName: '', Version: 3.8.10 (default, Sep 28 2021, 16:10:42) 
[GCC 9.3.0]) for '/home/cms-jovyan/Gaussian fit for different pt ranges.ipynb' (disableUI=true)
warn 04:46:06.114: Failed to get activated env vars for /bin/python3 in 1137ms
warn 04:46:06.114: Failed to get activated env vars for /bin/python3 in 1094ms
warn 04:46:06.114: Failed to get activated env vars for /bin/python3 in 1073ms
warn 04:46:06.114: Failed to get activated env vars for /bin/python3 in 1073ms
info 04:46:06.137: Process Execution: > /bin/python3 -c "import site;print("USER_BASE_VALUE");print(site.USER_BASE);print("USER_BASE_VALUE");"
> /bin/python3 -c "import site;print("USER_BASE_VALUE");print(site.USER_BASE);print("USER_BASE_VALUE");"
info 04:46:06.173: Process Execution: > /bin/python3 -m pip list
> /bin/python3 -m pip list
info 04:46:06.188: Process Execution: > /bin/python3 -c "import ipykernel; print(ipykernel.__version__); print("5dc3a68c-e34e-4080-9c3e-2a532b2ccb4d"); print(ipykernel.__file__)"
> /bin/python3 -c "import ipykernel; print(ipykernel.__version__); print("5dc3a68c-e34e-4080-9c3e-2a532b2ccb4d"); print(ipykernel.__file__)"
info 04:46:06.203: Process Execution: > /bin/python3 -m ipykernel_launcher --ip=127.0.0.1 --stdin=9003 --control=9001 --hb=9000 --Session.signature_scheme="hmac-sha256" --Session.key=b"a3ce49fd-0ae4-4152-b7e8-467c27d0e8a4" --shell=9002 --transport="tcp" --iopub=9004 --f=/home/cms-jovyan/.local/share/jupyter/runtime/kernel-v2-5193aWzm94dDDZjv.json
> /bin/python3 -m ipykernel_launcher --ip=127.0.0.1 --stdin=9003 --control=9001 --hb=9000 --Session.signature_scheme="hmac-sha256" --Session.key=b"a3ce49fd-0ae4-4152-b7e8-467c27d0e8a4" --shell=9002 --transport="tcp" --iopub=9004 --f=/home/cms-jovyan/.local/share/jupyter/runtime/kernel-v2-5193aWzm94dDDZjv.json
info 04:46:06.203: Process Execution: cwd: ~
cwd: ~
warn 04:46:06.214: Stderr output when getting ipykernel version & path Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'ipykernel' for /bin/python3
info 04:46:06.227: Process Execution: > /bin/python3 -c "import pip;print('6af208d0-cb9c-427f-b937-ff563e17efdf')"
> /bin/python3 -c "import pip;print('6af208d0-cb9c-427f-b937-ff563e17efdf')"
warn 04:46:06.228: StdErr from Kernel Process /bin/python3: No module named ipykernel_launcher

error 04:46:06.229: Disposing kernel process due to an error [zp [Error]: The kernel died. Error: /bin/python3: No module named ipykernel_launcher... View Jupyter [log](command:jupyter.viewOutput) for further details.
	at ChildProcess.<anonymous> (/home/cms-jovyan/.vscode-server/extensions/ms-toolsai.jupyter-2023.3.1000892223/out/extension.node.js:17:58325)
	at ChildProcess.emit (node:events:538:35)
	at Process.ChildProcess._handle.onexit (node:internal/child_process:291:12)] {
  category: 'kerneldied',
  kernelConnectionMetadata: da {
    kind: 'startUsingPythonInterpreter',
    kernelSpec: {
      specFile: '/home/cms-jovyan/.vscode-server/extensions/ms-toolsai.jupyter-2023.3.1000892223/temp/jupyter/kernels/pythonjvsc74a57bd0916dbcbb3f70747c44a77c7bcd40155683ae19c65e1c03b4aa3499c5328201f1/kernel.json',
      interpreterPath: '/bin/python3',
      isRegisteredByVSC: 'registeredByNewVersionOfExt',
      name: 'pythonjvsc74a57bd0916dbcbb3f70747c44a77c7bcd40155683ae19c65e1c03b4aa3499c5328201f1',
      argv: [Array],
      language: 'python',
      executable: 'python',
      display_name: 'Python 3',
      metadata: [Object],
      env: {}
    },
    interpreter: {
      id: '/bin/python3',
      sysPrefix: '/usr',
      displayPath: [E],
      envName: '',
      displayName: '',
      envType: 'Unknown',
      isCondaEnvWithoutPython: false,
      version: [Object],
      uri: [E]
    },
    id: '.jvsc74a57bd0916dbcbb3f70747c44a77c7bcd40155683ae19c65e1c03b4aa3499c5328201f1./bin/python3./bin/python3.-m#ipykernel_launcher'
  },
  exitCode: 1,
  stdErr: '/bin/python3: No module named ipykernel_launcher\n'
}
error 04:46:06.232: /bin/python3: No module named ipykernel_launcher

info 04:46:06.232: Dispose Kernel process 5293.
error 04:46:06.237: Kernel Launcher. launch, Class name = ah, completed in 1218ms, has a falsy return value, Arg 1: {"id":".jvsc74a57bd0916dbcbb3f70747c44a77c7bcd40155683ae19c65e1c03b4aa3499c5328201f1./bin/python3./bin/python3.-m#ipykernel_launcher","kernelSpec":{"specFile":"/home/cms-jovyan/.vscode-server/extensions/ms-toolsai.jupyter-2023.3.1000892223/temp/jupyter/kernels/pythonjvsc74a57bd0916dbcbb3f70747c44a77c7bcd40155683ae19c65e1c03b4aa3499c5328201f1/kernel.json","interpreterPath":"/bin/python3","isRegisteredByVSC":"registeredByNewVersionOfExt","name":"pythonjvsc74a57bd0916dbcbb3f70747c44a77c7bcd40155683ae19c65e1c03b4aa3499c5328201f1","argv":["python","-m","ipykernel_launcher","-f","{connection_file}"],"language":"python","executable":"python","display_name":"Python 3","metadata":{"interpreter":{"path":"/bin/python3"}},"env":{}},"interpreter":{"id":"/bin/python3","sysPrefix":"/usr","displayPath":"/bin/python3","envName":"","displayName":"","envType":"Unknown","isCondaEnvWithoutPython":false,"version":{"major":3,"minor":8,"patch":10,"raw":"3.8.10 (default, Sep 28 2021, 16:10:42) \n[GCC 9.3.0]"},"path":"/bin/python3"},"kind":"startUsingPythonInterpreter"}, Arg 2: 60000, Arg 3: "~/Gaussian fit for different pt ranges.ipynb", Arg 4: "~", Arg 5: <argument cannot be serialized for logging> [zp [Error]: The kernel died. Error: /bin/python3: No module named ipykernel_launcher... View Jupyter [log](command:jupyter.viewOutput) for further details.
	at ChildProcess.<anonymous> (/home/cms-jovyan/.vscode-server/extensions/ms-toolsai.jupyter-2023.3.1000892223/out/extension.node.js:17:58325)
	at ChildProcess.emit (node:events:538:35)
	at Process.ChildProcess._handle.onexit (node:internal/child_process:291:12)] {
  category: 'kerneldied',
  kernelConnectionMetadata: da {
    kind: 'startUsingPythonInterpreter',
    kernelSpec: {
      specFile: '/home/cms-jovyan/.vscode-server/extensions/ms-toolsai.jupyter-2023.3.1000892223/temp/jupyter/kernels/pythonjvsc74a57bd0916dbcbb3f70747c44a77c7bcd40155683ae19c65e1c03b4aa3499c5328201f1/kernel.json',
      interpreterPath: '/bin/python3',
      isRegisteredByVSC: 'registeredByNewVersionOfExt',
      name: 'pythonjvsc74a57bd0916dbcbb3f70747c44a77c7bcd40155683ae19c65e1c03b4aa3499c5328201f1',
      argv: [Array],
      language: 'python',
      executable: 'python',
      display_name: 'Python 3',
      metadata: [Object],
      env: {}
    },
    interpreter: {
      id: '/bin/python3',
      sysPrefix: '/usr',
      displayPath: [E],
      envName: '',
      displayName: '',
      envType: 'Unknown',
      isCondaEnvWithoutPython: false,
      version: [Object],
      uri: [E]
    },
    id: '.jvsc74a57bd0916dbcbb3f70747c44a77c7bcd40155683ae19c65e1c03b4aa3499c5328201f1./bin/python3./bin/python3.-m#ipykernel_launcher'
  },
  exitCode: 1,
  stdErr: '/bin/python3: No module named ipykernel_launcher\n'
}
error 04:46:06.238: Failed to connect raw kernel session: Error: The kernel died. Error: /bin/python3: No module named ipykernel_launcher... View Jupyter [log](command:jupyter.viewOutput) for further details.
warn 04:46:06.239: Error occurred while trying to start the kernel, options.disableUI=true [zp [Error]: The kernel died. Error: /bin/python3: No module named ipykernel_launcher... View Jupyter [log](command:jupyter.viewOutput) for further details.
	at ChildProcess.<anonymous> (/home/cms-jovyan/.vscode-server/extensions/ms-toolsai.jupyter-2023.3.1000892223/out/extension.node.js:17:58325)
	at ChildProcess.emit (node:events:538:35)
	at Process.ChildProcess._handle.onexit (node:internal/child_process:291:12)] {
  category: 'kerneldied',
  kernelConnectionMetadata: da {
    kind: 'startUsingPythonInterpreter',
    kernelSpec: {
      specFile: '/home/cms-jovyan/.vscode-server/extensions/ms-toolsai.jupyter-2023.3.1000892223/temp/jupyter/kernels/pythonjvsc74a57bd0916dbcbb3f70747c44a77c7bcd40155683ae19c65e1c03b4aa3499c5328201f1/kernel.json',
      interpreterPath: '/bin/python3',
      isRegisteredByVSC: 'registeredByNewVersionOfExt',
      name: 'pythonjvsc74a57bd0916dbcbb3f70747c44a77c7bcd40155683ae19c65e1c03b4aa3499c5328201f1',
      argv: [Array],
      language: 'python',
      executable: 'python',
      display_name: 'Python 3',
      metadata: [Object],
      env: {}
    },
    interpreter: {
      id: '/bin/python3',
      sysPrefix: '/usr',
      displayPath: [E],
      envName: '',
      displayName: '',
      envType: 'Unknown',
      isCondaEnvWithoutPython: false,
      version: [Object],
      uri: [E]
    },
    id: '.jvsc74a57bd0916dbcbb3f70747c44a77c7bcd40155683ae19c65e1c03b4aa3499c5328201f1./bin/python3./bin/python3.-m#ipykernel_launcher'
  },
  exitCode: 1,
  stdErr: '/bin/python3: No module named ipykernel_launcher\n'
}
error 04:46:06.244: interpreterPackages getPackageInformation, Class name = Kd, completed in 1268ms, has a falsy return value, Arg 1: {"interpreter":{"id":"/bin/python3","sysPrefix":"/usr","displayPath":{"$mid":1,"fsPath":"/bin/python3","path":"/bin/python3","scheme":"file"},"envName":"","uri":{"$mid":1,"fsPath":"/bin/python3","external":"file:///bin/python3","path":"/bin/python3","scheme":"file"},"displayName":"","envType":"Unknown","isCondaEnvWithoutPython":false,"version":{"major":3,"minor":8,"patch":10,"raw":"3.8.10 (default, Sep 28 2021, 16:10:42) \n[GCC 9.3.0]"}}} [su [Error]: Module 'pip' not installed.
	at au.execModule (/home/cms-jovyan/.vscode-server/extensions/ms-toolsai.jupyter-2023.3.1000892223/out/extension.node.js:7:27794)
	at processTicksAndRejections (node:internal/process/task_queues:96:5)
	at async Kd.getPackageInformation (/home/cms-jovyan/.vscode-server/extensions/ms-toolsai.jupyter-2023.3.1000892223/out/extension.node.js:7:19123)] {
  category: 'notinstalled',
  moduleName: 'pip'
}
info 04:46:32.558: Disposing kernel .jvsc74a57bd0916dbcbb3f70747c44a77c7bcd40155683ae19c65e1c03b4aa3499c5328201f1./bin/python3./bin/python3.-m#ipykernel_launcher for notebook /home/cms-jovyan/Gaussian fit for different pt ranges.ipynb due to selection of another kernel or closing of the notebook
info 04:46:32.558: Dispose Kernel '/home/cms-jovyan/Gaussian fit for different pt ranges.ipynb' associated with '/home/cms-jovyan/Gaussian fit for different pt ranges.ipynb'
info 04:46:48.972: Starting Jupyter Session startUsingRemoteKernelSpec, 452ef8d7cc1a94eff5970f8ca58f5742bc3dd4458f1635084009ac2e8058ea20..python3./opt/conda/python./.-m#ipykernel_launcher for '/home/cms-jovyan/Gaussian fit for different pt ranges.ipynb' (disableUI=false)
info 04:46:48.973: Checking for server existence.
info 04:46:48.974: Connecting to server
info 04:46:48.974: Connecting remote server kernel http://128.205.11.158:8882/
Connecting to Jupyter server at http://128.205.11.158:8882/
info 04:46:48.975: Creating server with url : http://128.205.11.158:8882/
info 04:46:48.978: Server started.
http://128.205.11.158:8882/: Kernel started: 0d439e92-db53-4a18-a4f9-070cdf73f842
info 04:46:49.820: Started session for kernel startUsingRemoteKernelSpec:452ef8d7cc1a94eff5970f8ca58f5742bc3dd4458f1635084009ac2e8058ea20..python3./opt/conda/python./.-m#ipykernel_launcher
http://128.205.11.158:8882/: Creating new notebook 
info 04:46:49.974: Checking for server existence.
info 04:46:49.980: Connecting to server
info 04:46:49.981: Creating server with url : http://128.205.11.158:8882/
info 04:46:49.986: Connecting remote server kernel http://127.0.0.1:8888/
Connecting to Jupyter server at http://127.0.0.1:8888/
info 04:46:49.986: Creating server with url : http://127.0.0.1:8888/
info 04:46:49.988: Server started.
info 04:46:49.988: Creating server with url : http://127.0.0.1:8888/
info 04:47:08.546: Cancel all remaining cells true || Error || undefined
info 04:48:25.987: Cancel all remaining cells true || Error || undefined
info 04:48:35.158: Cancel all remaining cells true || Error || undefined
info 04:48:35.314: Cancel all remaining cells true || Error || undefined
info 04:48:35.538: Cancel all remaining cells true || Error || undefined
info 04:48:47.127: Cancel all remaining cells true || Error || undefined
info 04:48:51.651: Cancel all remaining cells true || Error || undefined
info 04:49:26.908: Cancel all remaining cells true || Error || undefined
info 04:49:47.305: Cancel all remaining cells true || Error || undefined
info 04:49:51.587: Cancel all remaining cells true || Error || undefined
info 04:50:11.765: Cancel all remaining cells true || Error || undefined
info 04:50:14.269: Cancel all remaining cells true || Error || undefined
info 04:50:42.404: Cancel all remaining cells true || Error || undefined
info 04:51:02.539: Cancel all remaining cells true || Error || undefined
info 04:51:11.377: Restart kernel execution
info 04:51:11.378: Restart requested file:///home/cms-jovyan/Gaussian%20fit%20for%20different%20pt%20ranges.ipynb
info 04:51:38.172: Cancel all remaining cells true || Error || undefined
info 04:51:56.792: Cancel all remaining cells true || Error || undefined
info 04:52:02.594: Cancel all remaining cells true || Error || undefined
info 04:53:39.550: Cancel all remaining cells true || Error || undefined
info 04:55:05.940: Cancel all remaining cells true || Error || undefined
info 04:55:14.198: Cancel all remaining cells true || Error || undefined
